src/fsharp/FSharp.Compiler.nuget/Microsoft.FSharp.Compiler.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <PackageId>Microsoft.FSharp.Compiler</PackageId>
-    <PreRelease>false</PreRelease>
+    <PreRelease>true</PreRelease>
```

Comment:
Merge issue, I will fix it.

Reply:
Ahh … now I remember.  We used to build the compiler package twice, with and without the pre-release ID, this would allow us to release an RTM package to nuget.  However, we have since decided we don't need to put a release package on nuget with the new build stuff you are working on.  So we deleted the project for pre-release, and enabled pre-release here.